recipes/audioreach-rules/audioreach-rules/audioreach.rules

```diff
@@ -0,0 +1,2 @@
+SUBSYSTEM=="msm_audio_mem", KERNEL=="msm_audio_mem", GROUP="audio", MODE="0666", TAG+="uaccess"
```

Comment:
Why do you have MODE set to 0666? What are the 'msm_audio_mem' and 'aud_passthru_adsp' subsystems?